site/_apps/react-hn-stephen.txt

```diff
@@ -0,0 +1,27 @@
+---
+weight: 9
```

Comment:
@stephenkingsley I'm not pretty sure that the weight should be 15 in place of https://github.com/ragingwind/hacker-news-pwas/blob/3156b1edd2e33adbd254a455037af7a605fa6364/site/_apps/next-hnpwa.txt#L2